data/sql/updates/pending_db_world/rev_1625500923667306000.sql

```diff
@@ -0,0 +1,30 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1625500923667306000');
+
+/* Add new gossip */
+INSERT INTO `npc_text`(`ID`,`text0_0`,`text0_1`,`BroadcastTextID0`,
```

Comment:
please use `DELETE` before `INSERT` to make your queries re-executable (whenever possible)

Reply:
@FrancescoBorzi I added ```DELETE```. Could you take a look?